agot-bg-game-server/src/common/ingame-game-state/westeros-game-state/wildlings-attack-game-state/crow-killers-wildling-victory-game-state/CrowKillersWildlingVictoryGameState.ts

```diff
@@ -74,14 +172,16 @@ export default class CrowKillersWildlingVictoryGameState extends WildlingCardEff
     serializeToClient(admin: boolean, player: Player | null): SerializedCrowKillersWildlingVictoryGameState {
         return {
             type: "crow-killers-wildling-victory",
-            childGameState: this.childGameState.serializeToClient(admin, player)
+            childGameState: this.childGameState.serializeToClient(admin, player),
+            step: this.step
```

Comment:
@Longwelwind We don't need a migration for this new property, do we? It will be assigned with it's enum default `DEGRADING_KNIGHTS` which would then cause all current games in that state to perform the transformation in `onSelectUnitsEnd()`.

Reply:
Not really:

Line 184 will override the default value with `undefined` when the game is deserialized, so a migration is necessary.

Reply:
Thx. I will add the migration and set the step to it's default.